src/deviceApiCalls/transports/extension.transport.js

```diff
@@ -16,13 +18,21 @@ export class ExtensionTransport extends DeviceApiTransport {
 
     async send (deviceApiCall) {
         if (deviceApiCall instanceof GetRuntimeConfigurationCall) {
-            return deviceApiCall.result(await extensionSpecificRuntimeConfiguration(this.config))
+            return deviceApiCall.result(await extensionSpecificRuntimeConfiguration(this))
         }
 
         if (deviceApiCall instanceof GetAvailableInputTypesCall) {
             return deviceApiCall.result(await extensionSpecificGetAvailableInputTypes())
         }
 
+        if (deviceApiCall instanceof SetIncontextSignupDismissedAtCall) {
+            return deviceApiCall.result(await extensionSpecificSetIncontextSignupDismissedAt(deviceApiCall.params.value))
```

Comment:
Ultimately, we'll want to avoid explicitly calling these methods, the same way we do for other platforms. Because of that, I'd stick to passing the `params` on these calls, instead of `params.value`. This should make it easier to migrate to a more abstract solution that calls `deviceApiCall.method` with `deviceApiCall.params`, pretty much like [here](https://github.com/duckduckgo/duckduckgo-autofill/blob/3a329ee47224295fd4e8e85d8674daa126145134/src/deviceApiCalls/transports/apple.transport.js#L22).

Does it make sense?